revealjs_template.qmd

```diff
@@ -0,0 +1,117 @@
+---
+title: "<b>Title Slide<br>Max 4 Lines<br>Times New Roman (Heading)<br>40 Point</b>"
```

Comment:
I think the `_brand.yml` file is overriding the .scss formats. When I render the slides, this is not Times New Roman.

Reply:
Also weird for me - the text appears in TNR font from my end!